src/aggregation/bucket/histogram/histogram.rs

```diff
@@ -287,6 +296,12 @@ impl SegmentAggregationCollector for SegmentHistogramCollector {
 }
 
 impl SegmentHistogramCollector {
+    fn get_memory_consumption(&self) -> usize {
```

Comment:
we should really add byte-unit as a dependency of tantivy at one point... if you agree can you open a ticket?